metron-deployment/packaging/ambari/metron-mpack/pom.xml

```diff
@@ -60,6 +64,19 @@
                             <failIfNoMatch>false</failIfNoMatch>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>metron-short-version</id>
+                        <goals>
+                            <goal>regex-property</goal>
+                        </goals>
+                        <configuration>
+                            <name>metron.short.version</name>
+                            <value>${project.parent.version}</value>
+                            <regex>\.</regex>
```

Comment:
Does this still work if we have something like 0.3.0-SNAPSHOT? Or does the regex need to handle things like the '-'?